include/samplog/PluginLogger.h

```diff
@@ -72,6 +72,27 @@ namespace samplog
 		{
 			return samplog::LogNativeCall(m_Module.c_str(), amx, params, name, params_format);
 		}
+
+		inline bool operator()(const char *msg)
+		{
+			return Log(LogLevel::INFO, msg);
```

Comment:
I don't really like the idea of having a "default" log level. You can always create a custom log class which derives from the PluginLogger class as base and implement a default log function.